milestone2.ts

```diff
@@ -164,7 +165,10 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) =>
+  data.find(e => e.city === 'SLO' && e.age > 40) ? 
+  {...(data.find(e => e.city === 'SLO' && e.age > 40)) as Volunteer, position: "staff"} :
+  undefined; // Code here
```

Comment:
This function would look cleaner with multiple lines (especially since you have a duplicate find right now). To use multiple lines for a lambda function, just use curly braces {}
```
let daBigTest: GetVolunteer = (data) => {
  // code here, end with return
}
```